view/frontend/templates/iframe-handler.phtml

```diff
@@ -61,9 +58,31 @@ $scriptString = '
                 iframe.parentNode.insertBefore(divElement, iframe);
             });
         })(document, "iframe[data-cookieblock-src]", "marketing");
-    })
-    ';
-
+        
+        function fixIframeStyle () {
+            ((document, selector) => {
+                document.querySelectorAll(selector).forEach(iframe => {
+                    const pageBuilderContainer = iframe.closest(".pagebuilder-video-container");
+                    const pageBuilderColumn = iframe.closest(".pagebuilder-column");
+                    if (!pageBuilderContainer) {
+                        return;
+                    }
+                    if (!Cookiebot?.consent?.marketing) {
+                        pageBuilderContainer.style.paddingTop = "0";
+                        if (pageBuilderColumn) {
+                            pageBuilderColumn.style.backgroundColor = "#fff";
+                        }
+                    } else {
+                        pageBuilderContainer.style.paddingTop = "56.25%";
+                    }
+                });
+            })(document, "iframe", "marketing");
+        }
+        
+        addEventListener("CookiebotOnConsentReady", function() {
```

Comment:
This can be shortened to something like `addEventListener("CookiebotOnConsentReady", fixIframeStyle);`, right?